src/tpm.rs

```diff
@@ -151,8 +151,11 @@ pub fn is_vtpm() -> Option<bool> {
     match common::STUB_VTPM {
         true => Some(true),
         false => {
-            let tpm_manufacturer = get_tpm_manufacturer();
-            Some(tpm_manufacturer.unwrap() == "ETHZ")
+            if let Some(data) = get_tpm_manufacturer() {
```

Comment:
For cases where you can't eliminate the `else` clause (as here, because the `if` needs to yield a value), `match` is more clear; this should be:

```rust
match get_tpm_manufacturer() {
    Some(data) => Some(data == "ETHZ"),
    _ => Some(false).
};
```

Reply:
changed